inc/class-wpseo-shortlinker.php

```diff
@@ -58,4 +57,56 @@ public static function get( $url ) {
 	public static function show( $url ) {
 		echo esc_url( self::get( $url ) );
 	}
+
+	/**
+	 * Gets the current site's PHP version, without the extra info.
+	 *
+	 * @return string The PHP version.
+	 */
+	private function get_php_version() {
+		$version = explode( '.', PHP_VERSION );
+
+		return $version[0] . '.' . $version[1] . '.' . $version[2];
+	}
+
+	/**
+	 * Get our software and whether it's active or not.
+	 *
+	 * @return string The software name + activation state.
+	 */
+	private function get_software() {
+		$software = 'free';
+		if ( class_exists( 'WPSEO_Product_Premium' ) ) {
+			$software          = 'premium-inactive';
+			$product_premium   = new WPSEO_Product_Premium();
```

Comment:
If we have multiple shortlinks being created, this instance will be made every time. Probably it's nice to store this temporarily in a static variable.

```
    static $software;
    if ( $software === null ) {
       // Here the logic to determine the software.
    }

    return $software;  
```